tests/libtest.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Test if config exists.
+config_exists(){
+  local config_dir="/var/lib/container-storage-setup"
```

Comment:
I don't think library should hardcode the pathname where configurations are stored. If we need a funciton for this, library function should be passed in the path of directory where to search/look.